messages/de-DE.json

```diff
@@ -0,0 +1,384 @@
+      "undo": "Rückgängig machen"
+    },
+    "AddByURL": {
+      "button": "Mit URL hinzufügen",
+      "title": "Gruppe mit URL hinzufügen",
+      "description": "Wenn eine Gruppe mit dir geteilt wurde, kannst du ihre URL hier einfügen, um sie zu deiner Liste hinzuzufügen.",
+      "error": "Ups, wir können die Gruppe mit der angegebenen URL nicht finden…"
+    },
+    "NotFound": {
+      "text": "Diese Gruppe existiert nicht.",
+      "link": "Gehe zu zuletzt besuchten Gruppen"
+    }
+  },
+  "GroupForm": {
+    "title": "Gruppeninformationen",
+    "NameField": {
+      "label": "Gruppenname",
+      "placeholder": "Sommerurlaub",
+      "description": "Gib deiner Gruppe einen Namen."
+    },
+    "InformationField": {
+      "label": "Gruppeninformationen",
+      "placeholder": "Welche Informationen sind relevant für Gruppenmitglieder?"
+    },
+    "CurrencyField": {
+      "label": "Währungssymbol",
+      "placeholder": "$, €, £…",
+      "description": "Wir benutzen es, um Beträge anzuzeigen."
+    },
+    "Participants": {
+      "title": "Mitglieder",
+      "description": "Füge einen Namen für jedes Gruppenmitglied hinzu.",
+      "protectedParticipant": "Dieses Mitglied ist Teil der Ausgaben und kann nicht entfernt werden.",
+      "new": "Neu",
+      "add": "Mitglied hinzufügen",
+      "John": "Johannes",
+      "Jane": "Janina",
+      "Jack": "Jakob"
+    },
+    "Settings": {
+      "title": "Lokale Einstellungen",
+      "description": "Dies sind Einstellungen pro Gerät, die verwendet werden, um deine Nutzungsexperience zu verbessern.",
+      "ActiveUserField": {
+        "label": "Aktiver nutzer",
+        "placeholder": "Wähle ein Mitglied",
+        "none": "Keiner",
+        "description": "Standardnutzer, der die Ausgaben übernimmt."
+      },
+      "save": "Speichern",
+      "saving": "Speichert…",
+      "create": "Erstellen",
+      "creating": "Erstellt…",
+      "cancel": "Abbrechen"
+    }
+  },
+  "ExpenseForm": {
+    "Income": {
+      "create": "Einnahme erstellen",
+      "edit": "Einnahme bearbeiten",
+      "TitleField": {
+        "label": "Titel der Einnahme",
+        "placeholder": "Montagabend Restaurant",
+        "description": "Füge eine Beschreibung für die Einnahme hinzu."
+      },
+      "DateField": {
+        "label": "Datum der Einnahme",
+        "description": "Füge ein Datum hinzu für wann die Einnahme erhalten wurde."
+      },
+      "categoryFieldDescription": "Wähle die Kategorie der Einnahme.",
+      "paidByField": {
+        "label": "Empfangen von",
+        "description": "Wähle das Mitglied, das die Einnahme erhalten hat."
+      },
+      "paidFor": {
+        "title": "Empfangen für",
+        "description": "Wähle für wen die Einnahme empfangen wurde."
+      },
+      "splitModeDescription": "Wähle, wie die Einnahme aufgeteilt werden soll.",
+      "attachDescription": "Sehe und füge einen Beleg für die Einnahme hinzu."
```

Comment:
"Sehe und füge hinzu"? I don't know what that is supposed to mean.

Also "füge der Einnahme einen Beleg hinzu" sounds more elegant to me.